README.md

```diff
@@ -12,6 +12,7 @@ Parameters:
   - `template-path`: Path to the template file. Default: `.github/ISSUE_TEMPLATE/RELEASE.md`
   - `package-path`: Path to the package.json (used for next version number). Default: `package.json`
   - `moderators-path`: Optional path to the moderators file. Defaults to the bpmn-io moderators.
+  - `labels`: A comma-separated list of the labels you want to assign in the release issue.
```

Comment:
Why don't we make it a list?

```yaml
labels:
- release
- ready
```

Reply:
Nevermind, we cannot do it: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault